### DISCUSSION TITLE
EISS Donation Refund Requests

### DISCUSSION THREAD

**Discussion by moo1210** [Epix-Incorporated/Adonis#1323] [General]
It has come to our attention, that over 180+ and counting loyal Epix Inc customers have been disappointed in their EISS Donations
![image](https://github.com/Epix-Incorporated/Adonis/assets/27241938/c43d0887-b4d9-4fc1-9394-6f36c3ade290)

As such, we are offering a complimentary refund request process! Simply complete the following template:

> Roblox Username:
Link to which EISS Donation gamepass you bought: 
Reason for wanting a refund:

After completing the above template, an Epix Agent will get back to you as soon as possible. Please note a refund *request*, does not guarantee a refund.

**Comment by Dimenpsyonal** [comment]
Pls get EISS refund 😭 😭 😭 bad system 😭 ggive

**Comment by Expertcoderz** [comment]
Closing this due to a lack of responses for over a year.
